src/careamics/config/architectures/lvae_model.py

```diff
@@ -126,6 +124,13 @@ def validate_input_shape(cls, input_shape: list) -> list:
                 f"Input shape must be greater than 1 in all dimensions"
                 f"(got {input_shape})."
             )
+
+        if any(s < 64 for s in input_shape[-2:]):
+            raise ValueError(
+                f"Input shape must be greater or equal to 64 in XY dimensions"
+                f"(got {input_shape})."
+            )
+
         return input_shape
 
     @field_validator("encoder_n_filters")
```

Comment:
Validation could be refactored to include both parameters, that would mean changing the name of the variable in the validator and adapting the error message, as well as deleting the other validator.

```suggestion
    @field_validator("encoder_n_filters", "decoder_n_filters")
```